.rubocop_rspec_base.yml

```diff
@@ -104,7 +104,7 @@ Style/SignalException:
 Layout/SpaceAroundEqualsInParameterDefault:
   EnforcedStyle: no_space
 
-# We don't care about single vs double qoutes.
+# We don't care about single vs double quotes.
```

Comment:
Those are coming from rspec-dev, we have to ypdate there.

Reply:
rspec-rails is independently managed